File: Tecnicell.Server/Models/Responses/PhoneResponse.cs
using Tecnicell.Server.Models.Entity;
using Tecnicell.Server.Models.ViewModel;
using Tecnicell.Server.Models.ViewModel.Phone;

namespace Tecnicell.Server.Models.Responses
{
    public class PhoneResponse
    {
        public PhoneView? View { get; set; }
        public ImageViewModel? Image { get; set; }
        public IEnumerable<PhoneHistoryViewModel> Histories { get; set; } = new List<PhoneHistoryViewModel>();
    }
}

File: tecnicell.client/src/app/Interfaces/business/ApiResponses/PhoneResponse.ts
import { Image } from "../Models/Image";
import { PhoneHistory } from "../Models/Phone";

export interface PhoneResponse {
    view:      PhoneView;
    image:     Image;
    histories: PhoneHistory[];
}

export interface PhoneView {
    code:              string;
    type:              string;
    name:              string;
    salePrice:         number;
    imageCode:         string;
    cost:              number;
    currentState:      string;
    actionDescription: string;
}